tests/functional/cursor-001.phpt

```diff
@@ -14,35 +14,16 @@ $query = new MongoDB\Driver\Query(array(), array(
     'sort' => array('username' => 1),
 ));
 
-$qr = $manager->executeQuery(NS, $query);
-$cursor = $qr->getIterator();
-
-var_dump(
-    $cursor->getBatchSize(),
-    $cursor->setBatchSize(15),
-    $cursor->getBatchSize(),
-    $cursor->isDead()
-);
-
+$cursor = $manager->executeQuery(NS, $query);
 
 foreach ($cursor as $document) {
     echo $document['username'] . "\n";
 }
 
-var_dump(
-    $cursor->getBatchSize(),
-    $cursor->setBatchSize(15),
-    $cursor->getBatchSize(),
-    $cursor->isDead()
```

Comment:
The batchSize methods were removed, as we no longer allow this to be changed during iteration (we simply initialize it based on the Query's option). `isDead()` can be tested elsewhere.